Shuttle (New): Orca

assignedgod opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Added the Orca, a salvage ship to the frontier outpost shipyard

## Why / Balance
I wanted to try my hand at making a ship. I also quite enjoy salvage ships with exterior sections/open bays for throwing materials aboard, wanted one fit with a holopad, and I wanted it to be shaped like a fish. The idea for the design was an obsolete troop transport retrofitted for civilian use, with a large central bay (originally for holding troops) and a large open back for rapid deployment of troops, then swapped out for four zoned, open design areas within the larger area. In the front section, there are two maintenance rooms (atmos and a generator area), and a cockpit with access to two further exterior sections on the shoulder, the cockpit has several carp plushies and a holopad for communication purposes. 
I think it would be a good fit for frontier, mainly for the fact of it's ease of accsess, good storage space for a medium size ship. My inspiration was a cross between the sprinter and the gourd (Minus the science). I wanted a bigger cargo hub with a small medbay, an equipment zone, and a well defined cargo area on a ship much smaller than the gourd. 
I intend to, if this ship is accepted, modify it and make a pirate version afterwards.
## How to test
Simply buy the ship in game to test it, it is available from frontier shipyard

## Media
![image](https://github.com/user-attachments/assets/7d3e7b50-f55a-4e29-985c-6954209c3a69)
![image](https://github.com/user-attachments/assets/0c772493-d95e-407f-86aa-4387b5be2d3e)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->


**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Orca Ship to Frontier Shipyard
-->

arimah commented on Resources/Prototypes/_NF/Shipyard/orca.yml:
```diff
@@ -0,0 +1,34 @@
+- type: vessel
+  id: Orca
+  parent: BaseVessel
+  name: SV Orca
```
The "SV" prefix is reserved for shuttles in the scrapyard. Please choose another one. :)

assignedgod replied:
Oh shoot, I thought I'd changed it no NC, fixing

arimah commented on Resources/Prototypes/_NF/Shipyard/orca.yml:
```diff
@@ -0,0 +1,34 @@
+# Shuttle info! :3
```
Please replace this with a comment along this format:

```yml
# Author Info
# GitHub: (your GitHub username)
# Discord: (your Discord username)

# Maintainer Info
# GitHub: (your GitHub username)
# Discord: (your Discord username)

# Shuttle Notes:
#
```

You can leave Shuttle Notes empty if you have nothing to say.

assignedgod replied:
Okedokey, I'll update that!

arimah reviewed (changes requested):
Please include a screenshot of your ship. You should ideally have two screenshots:
- lights off (full brightness) with subfloor showing
- lights on with subfloor hidden

Please write a more detailed description that captures what the purpose of the ship is, why you think it would be a good fit for Frontier, what design considerations went into the ship, etc.

assignedgod commented:
> Please include a screenshot of your ship. You should ideally have two screenshots:
> 
>     * lights off (full brightness) with subfloor showing
> 
>     * lights on with subfloor hidden
> 
> 
> Please write a more detailed description that captures what the purpose of the ship is, why you think it would be a good fit for Frontier, what design considerations went into the ship, etc.

This should be all fixed up, I had a lot more written originally but I was worried about it being too wordy, I added that back in, added a little more context to what drove me to make the ship, and I have provided both requested screenshots, I hope it is to your liking :)

Cheackraze commented:
Hello there, the long awaited new guidelines are complete and can be found here: https://www.frontierstation14.com/ship-submission-guidelines I will be beginning reviews on all outstanding new ships starting this weekend. If you'd like to speed up the review process, be sure your submission adheres to the new guidelines!

assignedgod commented:
I'll be working on getting this up to the ship review standards and updating it today 👍

assignedgod commented:
So, this is looking like a tier 2 ship, with a primary salvage/mining area and secondary expanded cargo bay, it has two lively areas, namely a bunk and a small lounge. I am a little concerned with the design clarity, I like the blue tiles but it may have to go for the sake of visual clarity, but I wanted to see what others thought first
![image](https://github.com/user-attachments/assets/44245ec0-a812-4bb9-a0dd-ba00e9e20315)
![image](https://github.com/user-attachments/assets/6a753ab2-e3a2-4341-8617-211db65656bc)

Houtblokje commented:
> So, this is looking like a tier 2 ship, with a primary salvage/mining area and secondary expanded cargo bay, it has two lively areas, namely a bunk and a small lounge. I am a little concerned with the design clarity, I like the blue tiles but it may have to go for the sake of visual clarity, but I wanted to see what others thought first ![image](https://private-user-images.githubusercontent.com/79281911/420617493-44245ec0-a812-4bb9-a0dd-ba00e9e20315.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE0Njg4MDYsIm5iZiI6MTc0MTQ2ODUwNiwicGF0aCI6Ii83OTI4MTkxMS80MjA2MTc0OTMtNDQyNDVlYzAtYTgxMi00YmI5LWEwZGQtYmEwMGU5ZTIwMzE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA4VDIxMTUwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzNTYyOWUyMDlkNTU5ZDYyOGJiYjk1YjcwYWMwODQzZDBlY2JhNzI3OTc3ODU4YjUxNTA5NWNjZTRkMjAzNTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dyKvVo5E2i9m0AZv6wxWwxTgoue92rBq7kT45Qiakvk) ![image](https://private-user-images.githubusercontent.com/79281911/420617474-6a753ab2-e3a2-4341-8617-211db65656bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE0Njg4MDYsIm5iZiI6MTc0MTQ2ODUwNiwicGF0aCI6Ii83OTI4MTkxMS80MjA2MTc0NzQtNmE3NTNhYjItZTNhMi00MzQxLTg2MTctMjExZGI2NTY1NmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA4VDIxMTUwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQzYTZlYmI2YmU3YmM3NWU3ZjhjNTU0MGQ1ODY2ZTI2MzNiODA3MjczYjIwY2M5Y2I1ODEwM2UwODgyNjMwNzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.80wcnypBNH8yYjwFT7aqjjHxBZwRqT1NbjNOWulZTP0)

**I am not a maintainer, nor am I an experienced mapper, so please take my advice with a grain of salt**, but I personally think that your ship currently looks very overstimulating. I think what you could do is change some of the tiles to either the grey concrete or the grey tiles, and then use decals to seperate the different cargo areas into colors. That way, you could possibly still keep a bit of a blue accent, while making the ship look cleaner. The black/white tiled tiles you are using now are also more commonly associated with food ships/toilets, so you might want to change those to a more "clean" tile too. Near the bridge you have a few doors leading directly into space, which is spooky. 
Your choice of doors seem to indicate that this is a medical ship, while I think its supposed to be a salvage ship?

Overall, I really like the ship shape and the fact that it has quite a lot of room for some proper roleplaying, but the design language isn't very clear to me. Personally, if that gets cleaned up, I would use this ship as a cargo/salvaging ship, because the shape seems very unique and easy to use.

assignedgod commented:
I'll take those under consideration, I'll update soon with the new design.

assignedgod commented:
Closing this for now until I sit down and finish up on updating this. I wanna have the pirate version ready for it when I do.

Possumongus commented:
this is such a cute shaped ship

Houtblokje commented:
> Oh definitely. Part of my goal when designing it was making a ship that my pirate character Anne would fly, so first I had to make a base ship for it to mimic. Thus, We have the Orca, and soon we will have the pirate ship Fishy

I mean, technically you don't have to have a base ship for it to mimic, it could be another unique pirate ship.

assignedgod commented:
> > Oh definitely. Part of my goal when designing it was making a ship that my pirate character Anne would fly, so first I had to make a base ship for it to mimic. Thus, We have the Orca, and soon we will have the pirate ship Fishy
> 
> I mean, technically you don't have to have a base ship for it to mimic, it could be another unique pirate ship.

Indeed I don't, but flying a unique pirate ship is like announcing war ops- You get instantly spotted no matter what. We also desperately needed an alternative to the falcon as its become far, far too iconic

assignedgod commented:
![image](https://github.com/user-attachments/assets/53766089-80c6-479a-83ed-575a8539f4e7)
![image](https://github.com/user-attachments/assets/bf37b524-f565-4cd2-ad18-f0a62a66eb14)
![image](https://github.com/user-attachments/assets/2d85c4ad-96ec-4d7d-baa9-75263e9b5ae6)
![image](https://github.com/user-attachments/assets/524c7cb2-bb9f-4a1a-8045-1b467fc37539)

Yeah, so, closing it was unnecessary as I decided to just finish this today. I had wanted to add a pirate ship that could be a good alternative to the falcon, and thus, the fishy was born. It's designed to be a heavy hauling ship with combat potential that changes as more crew are added. Both ships are definitely a bit on the larger side, and I was unsure how I felt about that, but ultimately it would be very very hard to change and I wanted to just submit what I had, see what changes need to be done, and if it proved to be an issue I would change it then. I'm pretty happy with both, they feel good to fly and feel nice to walk around in. I am interested to see how they function on the server and with larger crews.

whatston3 commented on Resources/Prototypes/_NF/Shipyard/orca.yml:
```diff
@@ -0,0 +1,33 @@
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]
```
```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.
